R/utils.R

```diff
@@ -84,8 +84,9 @@ name_dots = function(...) {
 
 # convert a vector like c(1, 4, 3, 2) into a string like [1, 4, 3, 2]
 #   (common aggregation method for error messages)
-brackify = function(x) {
+brackify = function(x, quote=FALSE) {
   # arbitrary cutoff
+  if (quote && is.character(x)) x = paste0("'",head(x,11),"'")
```

Comment:
`head(x, 10L)` better, no?

Reply:
But then the next line doesn't add the `...`   The head(x,11) was to avoid doing the quoting on a huge 1m vector potentially.  11 just enough for the next line > 10 to trigger.

Reply:
ah of course. it's early... was thinking `>=` not `>`.